src/main/java/se/sundsvall/disturbance/service/message/SendMessageLogic.java

```diff
@@ -277,7 +278,7 @@ private void sendMessages(List<Message> messages) {
 		}
 	}
 
-	private CategoryConfig getMessageConfigByCategory(String category) {
-		return messageConfiguration.getCategoryConfig(Category.valueOf(category));
+	private Optional<CategoryConfig> getMessageConfigByCategory(final String category) {
+		return Optional.ofNullable(messageConfiguration.getCategoryConfig(Category.valueOf(category)));
```

Comment:
An alternative is to add `.orElse(new CategoryConfig())` to the end of this line, remove the Optional<> from method and remove `.orElse(new CategoryConfig())` from the places where this method is called, just to save some precious bytes...

Reply:
Yes, but the introducing of Optional was intended. This change was triggered by a SonaLift-issue.